.relint.yml

```diff
@@ -1,7 +1,5 @@
 - name: No ToDo
   pattern: "[tT][oO][dD][oO]"
   hint: Get it done right away!
-  filename:
-    - "*.py"
-    - "*.js"
+  filePattern: "((?!test_).).*\\.(py|js)"
```

Comment:
I'm not the regex master so forgive my silly question: shall this pattern exclude all files beginning with `test_`? If I run this in websites like [regexr](https://regexr.com/) I get this result:

![Screen Shot 2019-10-28 at 12 57 06](https://user-images.githubusercontent.com/1899950/67676755-82617300-f982-11e9-8456-f1c1efabc166.png)

Reply:
Good catch, there is a `^`missing in the beginning.